testservices/identityservice/userpass_test.go

```diff
@@ -56,7 +56,7 @@ var authTemplate = `{
 }`
 
 func userPassAuthRequest(URL, user, key string) (*http.Response, error) {
-	client := &http.DefaultClient
```

Comment:
Why? It's quite intentional to use a pointer here, instead of a copy.
If you needed to copy http.DefaultClient in order for the tests to pass, something is *wrong*.